cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -652,7 +656,8 @@
 
     public static final int DEFAULT_TIME_SERIES_TABLE_ROLL_TIME = 3600;
 
-    public static final long DEFAULT_RETENTION_HOURS = 2;
+    public static final long DEFAULT_SECOND_RETENTION_HOURS = 2;
+    public static final long DEFAULT_MIN_HOUR_RETENTION_DAYS = 30;
```

Comment:
these should be in cdap-default.xml, not constants in the code.

Reply:
Noticed they are already defined in cdap-default.xml, removed them